internal/tokens/fftokens/fftokens.go

```diff
@@ -716,16 +717,31 @@ func (ft *FFTokens) handleNamespaceStarted(ctx context.Context, data fftypes.JSO
 	// Make sure any pools that are marked as active in our DB are indeed active
 	namespace := data.GetString("namespace")
 	log.L(ctx).Debugf("Token connector '%s' started namespace '%s'. Ensuring all token pools active.", ft.Name(), namespace)
+
+	g, ctx := errgroup.WithContext(ctx)
```

Comment:
I feel like using `SetLimit() on the concurrency would be better than a completely unbounded parallelism against the token connector:
https://pkg.go.dev/golang.org/x/sync/errgroup#Group.SetLimit

Reply:
Good idea - will set that to the number of pools to activate